src/App/ComponentsDocumentation/components/Buttons/__snapshots__/index.test.js.snap

```diff
@@ -966,7 +1033,7 @@ exports[`Documentation: Buttons Overview renders 1`] = `
           },
           Object {
             "component": <Button
-              label="Do something"
+              label="Type something"
```

Comment:
I think "type something" is a bit awkward in this scenario (as it is a button)

Reply:
I second this. You're not typing something when you are clicking a button. Would it be possible to bring this up with the designers? Perhaps they have their own arguments.

Reply:
I totally agree. This is something I should have thought of myself. I will bring this up for the designers